contracts/migrations/9_transfer_ownership_and_tokens_for_mainnet.js

```diff
@@ -0,0 +1,47 @@
+const OriginToken = artifacts.require('./token/OriginToken.sol')
+const V00_Marketplace = artifacts.require('./V00_Marketplace.sol')
+
+const tokenMultiSig = '0xe011fa2a6df98c69383457d87a056ed0103aa352'
+const marketplaceMultiSig = '0x8a1a4f77f9f0eb35fb9930696038be6220986c1b'
+
+module.exports = function(deployer, network) {
+  return deployer.then(() => {
+    if (network === 'mainnet' || process.env['SIMULATE_MAINNET']) {
+      return transferTokensAndContractsToMultiSig()
```

Comment:
do we need an await here since transferTokensAndContractsToMultiSig is an async function otherwise truffle could mark this migration step as successful before it finishes to run ?

Reply:
truffle migrate runs the migrations [using 
 async.eachSeries](https://github.com/trufflesuite/truffle/blob/develop/packages/truffle-migrate/index.js#L194), so it'll wait for the promise to resolve